### ISSUE TITLE
[TextArea]: [In virtual PC mode, repetitive announcement for edit fields is observed]

### ISSUE THREAD

**Issue by AlexYanSap1** [SAP/ui5-webcomponents#13278]
### Describe the bug

1. click the sample StackBlitz page
2. open the Preview in the new tab
3. On this page, navigate through page with JAWS virtual PC mode (Insert + z) to read contents on page sequentially line by line using arrow key.
4. When edit fields are read, it is observed that here is repetitive announcement 

### Reproduction steps

see above

### Expected Behaviour

Repetitive announcement should be avoided as it cause a bad screen reader experience 

### Screenshots or Videos

<img width="1229" height="640" alt="Image" src="https://github.com/user-attachments/assets/14c37e6f-cef2-49df-b100-771750c7a707" />

### UI5 Web Components for React Version

2.20.0

### UI5 Web Components Version

2.20.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP SF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by hinzzx** [comment]
Hi @AlexYanSap1 ,

Could you please provide a link with the sample (or similar) that you were able to reproduce the issue with, so I could test and dispatch it to the correct team.

Thank you in advance,

Best,
Stoyan

**Comment by AlexYanSap1** [comment]
Hi @hinzzx ,

Sorry — I forgot to include the link earlier.

Here it is:
https://stackblitz.com/edit/github-szflcrvv?file=src%2FApp.tsx

**Comment by hinzzx** [comment]
Hi colleagues @UI5/ui5-team-ctr-ril ,

Could you please check the reported issue. The reporter provided sample above.

Best,
Stoyan

**Comment by niyap** [comment]
Hello @AlexYanSap1 ,

Thanks for sharing your findings!

Actually, this is the expected behavior. The “remaining characters” label visualised is announced first. Then, the second announcement comes from the <textarea> element, which has aria-label="500 characters remaining". This results in the full message: “500 characters remaining edit required.”
The aria-label attribute is necessary to ensure the remaining character count is announced when a virtual cursor is not being used.

Kind Regards,
Niya
